### PR TITLE
Add installation instructions to README

### PR DESCRIPTION
**Pull request by shanedewael**
* [x] I've read and understood the [Contributing guidelines](./CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
* [x] I've been mindful about doing atomic commits, adding documentation to my changes, not refactoring too much.
* [x] I've a descriptive title and added any useful information for the reviewer. Where appropriate, I've attached a screenshot and/or screencast (gif preferrably).
* [x] I've written tests to cover the new code and functionality included in this PR.
* [x] I've read, agree to, and signed the [Contributor License Agreement (CLA)](https://docs.google.com/a/slack-corp.com/forms/d/1q_w8rlJG_x_xJOoSUMNl7R35rkpA7N6pUkKhfHHMD9c/viewform).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,9 @@
...
 
-So you want to build a Slack app with Node.js? We've got you covered. `node-slack-sdk` is aimed at making
-building Slack apps ridiculously easy. This module will help you build on all aspects of the Slack platform,
-from dropping notifications in channels to fully interactive bots.
+This library is a wrapper around the Slack [RTM](https://api.slack.com/rtm) and [Web](https://api.slack.com/web) APIs. 
 
-This is a wrapper around the Slack [RTM](https://api.slack.com/rtm) and [Web](https://api.slack.com/web) APIs.
-
-This library provides the low level functionality you need to build reliable apps and projects on top of Slack's APIs.
+It will help you build on all aspects of the Slack platform, from dropping notifications in channels to fully interactive bots. It provides the low level functionality you need to build reliable apps and projects on top of Slack's APIs.
```
Comment (aoberoi) [inline README.md]: ehh, i don't agree with this. this module covers the Web API, RTM API (which you mention above) and Incoming Webhooks. it does not cover Events API, Interactive Messages, Outgoing Webhooks, or Slash Commands. want to try again?

File: README.md [hunk 2]
```diff
@@ -7,13 +7,9 @@
 
 Read the [full documentation](https://slackapi.github.io/node-slack-sdk) for all the lovely details.
 
-So you want to build a Slack app with Node.js? We've got you covered. `node-slack-sdk` is aimed at making
-building Slack apps ridiculously easy. This module will help you build on all aspects of the Slack platform,
-from dropping notifications in channels to fully interactive bots.
+This library is a wrapper around the Slack [RTM](https://api.slack.com/rtm) and [Web](https://api.slack.com/web) APIs.
```
Comment (aoberoi) [inline README.md]: prefer the word module to library, its more standard in node land

File: README.md [hunk 3]
```diff
@@ -27,6 +23,15 @@ Most Slack apps are interested in posting messages into Slack channels, and gene
...
 so we have them covered in [Building Bots](https://slackapi.github.io/node-slack-sdk/bots).
 
+# Installation
+Once you have a working Node.js project, you can install the Slack Developer Kit as a dependency via npm:
+
+```sh
+$ npm install @slack/client --save
+```
+
+If you are having troubles with setup and installation, check out our [full documentation](https://slackapi.github.io/node-slack-sdk/).
```
Comment (aoberoi) [inline README.md]: i don't think the full docs have much more than this (they are just more wordy)

File: README.md (hunk 1 above)
Comment (shanedewael) [inline README.md]: This verbiage was copied from the original README, but I will try to make more clear.

**Comment by shanedewael** [comment]
@aoberoi Made some changes based on your suggestions.

**Comment by aoberoi** [comment]
@shanedewael i like it! thanks 💯

**Pull request merged**